package-lock.json

```diff
@@ -705,17 +705,6 @@
         "underscore-contrib": "0.3.0"
       }
     },
-    "chai": {
```

Comment:
Why are there any package-lock.json changes with this PR?

Reply:
Now that I added a module I guess the `package-lock.json` changes are expected (before that, it was my mistake as a result of `git commit -a -m`)